macosx/InfoPeersViewController.mm

```diff
@@ -265,6 +243,67 @@ - (void)updateInfo
 
         self.fConnectedPeersField.stringValue = notActiveString;
     }
+    auto totalKnown = knownTracker + knownIncoming + knownCache + knownLpd + knownPex + knownDht + knownLtep;
+    NSString* knownText = [self connectedTextFrom:knownTracker:knownIncoming:knownCache:knownLpd:knownPex:knownDht:knownLtep];
+    if (totalKnown <= 1)
+    {
+        self.fConnectedPeersField.toolTip = [NSLocalizedString(@"Known:", "Inspector -> Peers tab -> peers")
+            stringByAppendingFormat:@" %@", totalKnown > 0 ? knownText : @"0"];
+    }
+    else
+    {
+        self.fConnectedPeersField.toolTip = [[NSString
+            localizedStringWithFormat:NSLocalizedString(@"%lu Known:", "Inspector -> Peers tab -> peers"), totalKnown]
+            stringByAppendingFormat:@" %@", knownText];
+    }
+}
+
+- (NSString*)connectedTextFrom:(NSUInteger)tracker //
```

Comment:
nit: Could it be class method?

Reply:
next time, since it's now merged. :)
Or feel free to PR --++